### PR TITLE
File changes update project updated_at

### PR DESCRIPTION
**Pull request by DylanTinianov**
Closes: #126 

## Description

- Adding, updating, or deleting files will now also update the project in order to set the new updated_at time.

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: controller/files.go [hunk 1]
```diff
@@ -74,6 +84,33 @@ func (f *Files) DeleteFile(scriptID, projectID uuid.UUID) error {
...
+	if err != nil {
+		return err
+	}
+
+	err = f.store.UpdateProject(model.UpdateProject{
+		ID:          proj.ID,
+		Title:       nil,
+		Description: nil,
+		Readme:      nil,
+		Persist:     nil,
```
Comment (sideninja) [inline controller/files.go:108]: Why is this set to nil? I feel you should just skip all these. Also are you doing this to just update the updated_at time? A comment as to why this is done is really needed here.

File: controller/files.go [hunk 2]
```diff
@@ -74,6 +84,33 @@ func (f *Files) DeleteFile(scriptID, projectID uuid.UUID) error {
...
+	}, &proj)
+	if err != nil {
+		return err
+	}
+
 	return nil
```
Comment (sideninja) [inline controller/files.go:114]:
```suggestion
    return err
```

Comment (sideninja) [reply controller/files.go:114]: Or even just `return f.store.UpdateProject`

**Review by sideninja — CHANGES_REQUESTED** [review]
Left some comments

File: controller/files.go (hunk 1 above)
Comment (DylanTinianov) [reply controller/files.go]: Cleaned this up and added better comments

**Review by sideninja — APPROVED** [review]
Nice work!

**Pull request merged**